### ISSUE TITLE
Feature request: for linux

### ISSUE THREAD

**Issue by Gittalax** [KamaleiZestri/BinaryReceptor#1]
Great, thanks! This is a long-awaited little miracle. It is a pity that only for windows. I only use pure linux. I hope that in the future it will be possible to write it on ELF 64-bit lsb.

Markus have mentioned before the http/netcat submit method via LAN. Unfortunately it is not applicable on our large and pomiscuous wireless network. And in the most cases, while scanning the inventory, the only accessible network is the mobile 4G. Well, indeed, bluetooth HID to the linux notebook would be the coveted method.

**Comment by KamaleiZestri** [comment]
Yeah, I had a feeling a linux version would be needed eventually. I will start working on that.
